### PR TITLE
Python native

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: recipes-debian/python/python-native_debian.bb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  STAGING_LIBDIR=${STAGING_LIBDIR_NATIVE} \
+  STAGING_INCDIR=${STAGING_INCDIR_NATIVE} \
+'
+
+do_debian_patch_prepend() {
+    export DEB_TARGET_ARCH=${DPKG_ARCH}
+    export DEB_TARGET_ARCH_OS=${TARGET_OS}
+
+    cd ${DEBIAN_UNPACK_DIR}
+    chmod +x ./debian/rules
```
Comment (masami256) [inline recipes-debian/python/python-native_debian.bb]: @yuwata  rules file is kind of makefile, so you can execute this file via make command.

File: recipes-debian/python/python-native_debian.bb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    # Generage ./debian/patches/series
+    ./debian/rules ./debian/patches/series
+}
+
+do_debian_patch_append() {
```
Comment (masami256) [inline recipes-debian/python/python-native_debian.bb]: How about remove multiarch-libc.diff from ./debian/patches/series after generate debian/patches/series file in do_debian_patch_prepend() ?

**Comment by yuwata** [comment]
@masami256 Thank you for the review. Updated. PTAL.

**Comment by masami256** [comment]
thanks! LGTM.

**Pull request merged**